src/main/java/fi/otavanopisto/kuntaapi/server/controllers/ServiceController.java

```diff
@@ -63,18 +63,19 @@ public Service findService(ServiceId serviceId) {
     return null;
   }
   
-  public List<Service> listServices(OrganizationId organizationId, Long firstResult, Long maxResults) {
-    List<Service> result = new ArrayList<>();
+  public SearchResult<Service> searchServices(OrganizationId organizationId, ElectronicServiceChannelId electronicServiceChannelId, PhoneServiceChannelId phoneServiceChannelId, PrintableFormServiceChannelId printableFormServiceChannelId, ServiceLocationServiceChannelId serviceLocationServiceChannelId, WebPageServiceChannelId webPageServiceChannelId, String search, ServiceSortBy sortBy, SortDir sortDir, Long firstResult, Long maxResults) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 11 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)